package.json

```diff
@@ -1,6 +1,22 @@
 {
   "name": "highchartTable",
   "version": "1.1.0",
+  "description": "This jQuery plugin provides a simple way to convert HTML data tables to Highcharts graphs.",
+  "author": "agallou",
```

Comment:
there is multiple contributors to the project. Could you change the author to "highchartTable Development Team (https://github.com/highchartTable/jquery-highchartTable-plugin/graphs/contributors)" ?